backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/MetricsHandler.scala

```diff
@@ -246,6 +246,8 @@ class MetricsHandler extends MetricsApi with Logging {
       "compressTime" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime to compress"),
       "prepareTime" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime to prepare"),
       "decompressTime" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime_decompress"),
+      "ipcTime" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime_ipc"),
+      "deserializeTime" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime_deserialize"),
```

Comment:
Should we use the consistent metric name format?
e.g. `totaltime_XXX` or `totaltime XXX`

cc @rui-mo

Reply:
it's good to rename to `totaltime XXX` consistently.

Reply:
Yes, we'd better keep them algined.